### PR TITLE
Fix spelling typos

### PR DESCRIPTION
**Pull request by Aniak5**

### CODE REVIEW

**Review by rroupski — APPROVED** [review]
Thanks!

**Review by floydtree — APPROVED** [review]

**Review by mr-splunk — APPROVED** [review]
Looks good

File: objects/idp.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
       "description":"The name of the identity provider."
```
Comment (pagbabian-splunk) [inline objects/idp.json:8]: idp.json is now under the entities folder and extends entity.

Comment (Aniak5) [reply objects/idp.json:8]: hmmm thats weird, let me check and make sure everything was merged in properly

Comment (Aniak5) [reply objects/idp.json:8]: seems like everything was properly merged

**Pull request merged**